packages/rest-hooks/src/state/PollingSubscription.ts

```diff
@@ -32,8 +35,15 @@ export default class PollingSubscription implements Subscription {
     this.url = url;
     this.frequencyHistogram.set(this.frequency, 1);
     this.dispatch = dispatch;
-    if (isOnline()) this.update();
-    this.run();
```

Comment:
Hmm, doesn't this.run() still need to happen?

Reply:
It happens now by calling this.onlineListener() if we are online on line 43